.github/workflows/generate-permit.yml

```
@@ -0,0 +1,37 @@
+name: Generate Permit
+
+on:
+  workflow_dispatch:
+    inputs:
+      beneficiary:
+        required: true
+        description: The address of the beneficiary's wallet
+      ubiquibot:
```

```suggestion
      evmPrivateEncrypted:
```

This would match our config property name of `keys.evmPrivateEncrypted`